src/rest-server/src/models/v2/virtual-cluster/k8s.js

```diff
@@ -138,18 +160,31 @@ const getVcList = async () => {
   }
 
   // set configured resource
-  for (let vc of Object.keys(vcData.virtualCellCapacity)) {
-    vcInfos[vc].resourcesTotal.memory = vcData.virtualCellCapacity[vc].resourcesTotal.memory;
-    vcInfos[vc].resourcesTotal.cpu = vcData.virtualCellCapacity[vc].resourcesTotal.cpu;
-    vcInfos[vc].resourcesTotal.gpu = vcData.virtualCellCapacity[vc].resourcesTotal.gpu;
+  if (launcherConfig.enabledHived) {
+    for (let vc of Object.keys(vcData.virtualCellCapacity)) {
+      vcInfos[vc].resourcesTotal.memory = vcData.virtualCellCapacity[vc].resourcesTotal.memory;
+      vcInfos[vc].resourcesTotal.cpu = vcData.virtualCellCapacity[vc].resourcesTotal.cpu;
+      vcInfos[vc].resourcesTotal.gpu = vcData.virtualCellCapacity[vc].resourcesTotal.gpu;
+    }
+  } else {
+    const nodes = (await axios({
+      method: 'get',
+      url: `${kubernetesConfig.apiserver.uri}/api/v1/nodes`,
+    })).data.items.filter((node) => node.metadata.labels['pai-worker'] === 'true');
```

Comment:
Will this be called only once at init or every time refresh the VC page?

Reply:
same